index.css

```diff
@@ -71,3 +75,81 @@ hr {
 #data tr:nth-child(odd) {
   background-color: #fff;
 }
+
+#navBar {
+  box-shadow: 3px 3px 5px rgba(0, 0, 0, 0.3);
+  border: 2px solid #000;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 15px;
+}
+
+#navUl {
+  padding-left: 30px;
+  padding-right: 50px;
+}
+
+#navUl > li {
+  display: inline-block;
+  margin-left: 20px;
+  border-left: 2px solid black;
+  padding-left: 40px;
+  padding-right: 40px;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+#navUl > li:first-child {
+  border-left: none;
+}
+
+#logo {
+  display: inline-block;
+}
+
+.t16 {
+  font-size: 18px;
+}
+
+.lstClass {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 100px;
+}
+
+.noLstClass {
+  display: none;
+}
+
+.nobookList {
+  display: none;
+}
+
+.cntSection {
+  margin-top: 40px;
+}
+
+footer {
+  padding: 20px;
+  box-shadow: 3px 3px 5px rgba(0, 0, 0, 0.3);
+  border: 2px solid #000;
+  margin-top: 100px;
+}
```

Comment:
- [ ] Great job so far, you have done really well, I think we can improve this one thing, if you notice in the wireframe the footer is at the bottom in all three sections, however, in your design it is not fixed to the bottome, please try to fix this :+1: